auth/src/main/java/org/aerogear/auth/CredentialsType.java

```diff
@@ -0,0 +1,5 @@
+package org.aerogear.auth;
+
+public enum CredentialsType {
+    TOKEN, BROWSER;
```

Comment:
not sure those values make sense for `credentials`. 

To make it clear, I think one of the type can be "OpenConnectId". It will be the only one we are going to support for now.